docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -137,68 +136,146 @@ To output predicted class probabilities instead of predicted classes, you can us
+
+```{.python .input}
+y_pred = predictor.predict(test_data_nolabel)
 predictor.evaluate_predictions(y_true=y_test, y_pred=y_pred, auxiliary_metrics=True)
 ```
 
 However, you must be careful here as certain metrics require predicted probabilities rather than classes.
-Since the label columns remains in the `val_data` DataFrame, we can instead use the shorthand:
+Since the label columns remains in the `test_data` DataFrame, we can instead use the shorthand:
 
-```
-predictor.evaluate(val_data)
+```{.python .input}
+predictor.evaluate(test_data)
 ```
 
 which will correctly select between `predict()` or `predict_proba()` depending on the evaluation metric.
 
+## Interpretability (feature importance)
+
+To better understand our trained predictor, we can estimate the overall importance of each feature:
+
+```{.python .input}
+importance_scores = predictor.feature_importance(test_data)
+print(importance_scores)
+```
+
+Computed via [*permutation-shuffling*](https://explained.ai/rf-importance/), these feature importance scores quantify the drop in predictive performance (of the already trained predictor) when one column's values are randomly shuffled across rows. The top features in this list contribute most to AutoGluon's accuracy (for predicting when/if a patient will be readmitted to the hospital). Features with non-positive importance score hardly contribute to the predictor's accuracy (at least not on an individual basis, these features may still provide useful predictive signal through interaction-effects with other features). These scores facilitate interpretability of the predictor's global behavior (which features it relies on for *all* predictions) rather than [local explanations](https://christophm.github.io/interpretable-ml-book/taxonomy-of-interpretability-methods.html) that only rationalize one particular prediction.
+
+
+## Accelerating inference
+
+We describe multiple ways to reduce the time it takes for AutoGluon to produce predictions.
 
-## Maximizing predictive performance
+### Keeping models in memory
 
-To get the best predictive accuracy with AutoGluon, you should generally use it like this:
+By default, AutoGluon loads models into memory one at a time and only when they are needed for prediction. This strategy is robust for large stacked/bagged ensembles, but leads to slower prediction times. If you plan to repeatedly make predictions (e.g. on new datapoints one at a time rather than one large test dataset), you can first specify that all models should be loaded into memory as follows:
 
 ```{.python .input}
-long_time = 60 # for quick demonstration only, you should set this to longest time you are willing to wait
-predictor = task.fit(train_data=train_data, label=label_column, eval_metric=metric, auto_stack=True, time_limits=long_time)
+predictor._learner.persist_trainer() # load models into memory (still experimental, may fail for big ensembles)
```

Comment:
May wish to wait until #640 is merged and base this section on that functionality.

Reply:
replaced with `predictor.persist_models()`